config/prod/JKGRinst.yaml

```diff
@@ -0,0 +1,47 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v10.j2
```

Comment:
this is not the latest template.  please take another look at the example template

Reply:
Changed to remote/managed-ec2-linux-v1.j2